plumbum/commands/base.py

```diff
@@ -335,7 +335,7 @@ def popen(self, args=(), **kwargs):
 
 
 class BoundEnvCommand(BaseCommand):
-    __slots__ = ("cmd",)
+    __slots__ = ("cmd", "env", "cwd")
```

Comment:
Mistake? "env" and "cwd" are already in `BaseCommand` `__slots__`.

Reply:
probably.
won't this override? or you're saying each base class keeps its own `__slots__`?